src/kvirc/kernel/KviIrcLink.cpp

```diff
@@ -258,14 +258,14 @@ void KviIrcLink::processData(char * buffer, int iLen)
 
 			// FIXME: actually it can happen that the socket gets disconnected
 			// in a incomingMessage() call.
```

Comment:
comments user never sees Im split on these, if search and replace finds them... else I dont care much